src/main/webapp/css/main.css

```diff
@@ -129,11 +129,9 @@ button {
   overflow-y: scroll;
 }
 
-/* This is for the Search Results Page */
-#results {
```

Comment:
We not using this anymore or removed by mistake?

Reply:
Not using it anymore. It was from when the search bar redirected to a results page.